PROTOCOL.md

```diff
@@ -95,6 +96,19 @@ By default, the reference implementation stores data files in directories that a
 This directory format is only used to follow existing conventions and is not required by the protocol.
 Actual partition values for a file must be read from the transaction log.
 
+### Change Data Files
+Change data files are stored in a directory at the root of the table named `_change_data`, and represent the changes for the table version they are in. For data with partition values, change data files are stored within the `_change_data` directory in their respective partitions (i.e. `_change_data/part1=value1/...`). Writers can _optionally_ produce these change data files as a consequence of operations that change underlying data, like `UPDATE`, `DELETE`, and `MERGE` operations to a Delta Lake table. When available, change data readers should use the change data files instead of computing changes from the underlying data files.
```

Comment:
the reference implementation does not write change files for append-only - should we explicitly call this out here?

Reply:
btw, in case you haven't seen it, here is the public design doc: https://docs.google.com/document/d/1FQSl0S0Irckim331YvcAKyxxGBhdLiJcJH6v3fEyiC0/edit#

taken from the public github issue. https://github.com/delta-io/delta/issues/1105

> should we explicitly call this out here

I think so.